jena-fuseki2/jena-fuseki-geosparql/src/main/java/org/apache/jena/ext/io/github/galbiston/rdf_tables/file/FileConverter.java

```diff
@@ -101,6 +101,7 @@ public static void writeToModel(File inputFile, Model model, char delimiter, Boo
         } catch (IOException | RuntimeException | CsvValidationException ex) {
 
             LOGGER.error("FileConverter: Line - {}, File - {}, Exception - {}", lineNumber, inputFile.getAbsolutePath(), ex.getMessage());
+            ex.printStackTrace();
```

Comment:
Currently debugging it? Or if intended to leave in the code maybe better to be wrapped in another exception thrown?

Reply:
Good point. Moved the exception into the `AssertionError`. Previously, it was hiding the information.